support/ocm.go

```diff
@@ -69,10 +69,26 @@ func GetMachinePools(t Test, connection *ocmsdk.Connection) []*cmv1.MachinePool
 	return MachinePools(t, connection)(t)
 }
 
-func MachinePoolId(machinePool *cmv1.MachinePool) string {
```

Comment:
This function is used in e2e tests - https://github.com/project-codeflare/codeflare-operator/blob/main/test/e2e/instascale_machinepool_test.go#L55

Reply:
Yes sorry, the refactor of this test to check for labels instead is in an unmerged PR so this is currently being used. I will revert, thanks.